pkg/operator/controller/nodeobservability/rbac.go

```diff
@@ -84,6 +81,14 @@ func (r *NodeObservabilityReconciler) desiredClusterRole(nodeObs *v1alpha1.NodeO
 			Name:      clusterRoleName,
 			Namespace: nodeObs.Namespace,
 			Labels:    labelsForClusterRole(clusterRoleName),
+			OwnerReferences: []metav1.OwnerReference{
+				{
+					Name:       nodeObs.Name,
+					Kind:       nodeObs.Kind,
+					UID:        nodeObs.UID,
+					APIVersion: nodeObs.APIVersion,
+				},
+			},
```

Comment:
@lmzuccarelli @josefkarasek 
Do you think we should keep this? 
It normally doesnt have any effect to set ownership on cluster scoped resources.
Let me know 
Applicable to SCC and clusterrolebinding as well

Reply:
If it has no effect then delete it - less code is always better :)

Reply:
OK done